src/css/flags/controls-disabled.less

```diff
@@ -10,4 +10,8 @@
     .jw-media {
         cursor: auto;
     }
+
```

Comment:
You're right, fixing now. Thought I tested it in Firefox, but must have been Safari.